script.js

```diff
@@ -12,3 +14,136 @@ menu.forEach((menuBtn) => {
+    }
+    card.innerHTML = `
+      <div class="works-image">
+        <img src="${portfolioDetails[i].img}" alt="Multi-post Stories">
+      </div>
+      <div class="details d-flex">
+        <h2 class="works-header">${portfolioDetails[i].name}</h2>
+        <ul class="header-sub d-flex">
+          <li class="main-header">${portfolioDetails[i].title}</li>
+          <li><img src="img/Counter.png" alt="Counter"></li>
+          <li>${portfolioDetails[i].title}</li>
+          <li><img src="img/Counter.png" alt="Counter"></li>
+          <li>2015</li>
+        </ul>
+        <p class="works-text">${portfolioDetails[i].text}</p>
+        <ul class="works-stack d-flex">
+          ${portfolioDetails[i].workStack.map((stack) => `<li>${stack}</li>`).join('')}
+        </ul>
+        <button class="works-button portfolio-details">See Project</button>
+      </div>
+    `;
+    portfolio.appendChild(card);
+  }
+}
+
+const portfolioItems = document.querySelectorAll('.portfolio-details');
+if (portfolioItems) {
+  portfolioItems.forEach((cards) => {
+    cards.addEventListener('click', (e) => {
+      e.preventDefault();
+      const idValue = e.target.closest('.tonic').id;
+      for (let i = 0; i < portfolioDetails.length; i += 1) {
+        if (portfolioDetails[i].id === idValue) {
+          const popupSection = document.createElement('section');
+          popupSection.classList.add('popup-section');
+          popupSection.innerHTML = `
+              <div class="background"></div>
+              <div class="tonic d-flex">
+                <button class="close" type="submit">
+                  <i class="bi bi-x"></i>              
+                </button>
+                <h2 class="works-header">${portfolioDetails[i].name}</h2>
+                <ul class="header-sub d-flex">
+                  <li class="main-header">${portfolioDetails[i].title}</li>
+                  <li><img src="img/Counter.png" alt="Counter"></li>
+                  <li>${portfolioDetails[i].stack}</li>
+                  <li><img src="img/Counter.png" alt="Counter"></li>
+                  <li>2015</li>
+                </ul>
+                <div class="works-image" style="background-image: url(${portfolioDetails[i].img});">
+                </div>
+                <div class="details d-flex">
+                  <p class="works-text">${portfolioDetails[i].text}</p>
+                  <ul class="works-stack d-flex">
+                    ${portfolioDetails[i].workStack.map((stack) => `<li>${stack}</li>`).join('')}
+                  </ul>
+                  <div class="d-flex justify-content-between link-buttons">
+                    <button class="works-button"><a href="${portfolioDetails[i].live}">See Live <img src="img/live.png" alt="Live Link"/></a></button>
+                    <button class="works-button"><a href="${portfolioDetails[i].source}">See Source <img src="img/git.png" alt="Source Link"/></a></button>
```

Comment:
[OPTIONAL] You have done a great job implementing the functionality of the buttons that show up on the pop-up window 👍. I think that it will greatly improve the User Experience of the page if the buttons don't take the user away from the page but creates a new tab and opens the attached links in those tabs 💪. For example, if I should click on the ``` See Source ``` button, a new tab would be opened to display the linked GitHub repository. To do this you would just have to add the ```target``` attribute to the embedded link tags and give the attribute a value of ```_blank``` 😉. In the end, your code may look something like this: 

```sh
<button class="works-button">
   <a href="${portfolioDetails[i].live}" target="_blank">See Live <img src="img/live.png" alt="Live Link"/></a>
</button>
<button class="works-button">
   <a href="${portfolioDetails[i].source}" target="_blank">See Source <img src="img/git.png" alt="Source Link"/></a>
</button>
```